server/app/services/geo/esri/RealEsriClient.java

```diff
@@ -233,6 +233,12 @@ private WSRequest createWebRequest(String url, ObjectNode addressJson) {
     // session
     request.addQueryParameter("maxLocations", "3");
 
+    // The forStorage parameter specifies whether the results of the operation will be persisted.
+    // The default value is false, which indicates the results of the operation can't be stored, but
+    // they can be temporarily displayed on a map, for instance. If you store the results, in a
+    // database, for example, you need to set this parameter to true.
```

Comment:
This is lifted directly from esri's documentation. There's no direct link anchor to jump to. The link to the docs is already in the class level javadoc.